definitions/npm/ramda_v0.26.x/flow_v0.104.x-/ramda_v0.26.x.js

```diff
@@ -777,11 +777,15 @@ declare module ramda {
     fn: UnaryPredicateFn<T>,
   ): (xs: Array<T>) => boolean;
 
-  declare function nth<V, T: Array<V>>(i: number, xs: T): ?V;
-  declare function nth<V, T: Array<V> | string>(
-    i: number,
-  ): ((xs: string) => string) & ((xs: T) => ?V);
-  declare function nth<T: string>(i: number, xs: T): T;
+  declare var nth: {
+    <A, As: $ReadOnlyArray<A>>(n: number, xs: As): ?A,
+    <A, As: $ReadOnlyArray<A> | string>(
+      n: number,
+    ): ((xs: string) => string) & ((xs: As) => ?A),
+    (n: number, xs: string): string,
+    ...
```

Comment:
Unless this causes some tests to fail, I would remove the inexact notation here and make it exact.

Reply:
It is causing tests to fail on flow v104 and onward, I added the inexact operator as the test report prompted me to, but let's use the exact type instead.

Reply:
If it causes failure I'm fine letting it go for now. The Flow split can be handled later.